### ISSUE TITLE
rename "resolved" status to "closed" or "done"

### ISSUE THREAD

**Issue by Dieterbe** [naggie/dstask#89]
i propose to rename the status "resolved" to "done" (or possibly "closed")
for 2 reasons:
1) "resolved" is rather ambiguous. "done" and "closed" are more clear and self-evident.
2) brief words are good. "resolved" is needlessly long.

so then, done or closed? depends, do we ever envision closing ("resolving") a task without actually doing the work?
if yes, "closed" seems more approriate. we could potentially even have both, to distinguish both cases.

in fact, we already use the term "done" in a variety of places, e.g.
```
$ task help
Available commands:

next              : Show most important tasks (priority, creation date -- truncated and default)
add               : Add a task
(...)
done              : Resolve a task
(...)
```

**Comment by naggie** [comment]
I deliberately avoided "done" as a task might be concluded in but not actioned. "Closed" is probably as good. Not convinced it's worth a migration or backwards-compatible code. @dontlaugh what do you think?

**Comment by dontlaugh** [comment]
I don't think this is important enough to change right now. But if there's a reason to create/test a migration strategy, we can perhaps introduce this change after that.

I agree that the language is ambiguous. "Closed" is arguably less ambiguous.
